app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] I did look into your `DB` folder and they are a bit different, you are not referencing the `Author` and it should refer to class_name user.

Reply:
Done